index.html

```diff
@@ -7,10 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:wght@200&display=swap" rel="stylesheet"> 
 </head>
-
 <body>
     <!-- Add your markup here -->
-</body>
-
+    <header class="header-container">
+        <div class="logo">
+            <img 
+            src="/img/mint-cupcake-logo2.png" 
+            alt="logo"
+            width="100%"
+            height="100%">
+        </div>
+        <div class="nav-bar">
+                <label for="toggle">&#9776;</label>
+                <input type="checkbox" id="toggle"/>
+                <div class="menu">
```

Comment:
You could use a `nav` tag here instead of `div` to make it more semantic

Reply:
Thanks Zein, I made it with nav, but I got help here and she helped me like that. it was in my mind, I'll return it to nav.